packages/react-router/docs/api/Route.md

```diff
@@ -178,3 +178,11 @@ When `true`, a `path` that has a trailing slash will only match a `location.path
 | `/one` | `/one` | yes |
 | `/one` | `/one/` | no |
 | `/one` | `/one/two` | no |
+
+## location: object
+
+A `<Route>` element tries to match its `path` to the current browser `location`.
+However, a [`location`](location.md) object with a different `pathname` can be passed for matching, instead of the current browser location.
```

Comment:
The term *pathname* may cause ambiguity.
It felt necessary to keep things verbose to prevent confusion with "location" being inferred as the URL.

> However, a location with a different pathname

> However, a location object with a different pathname

---

Although, one may argue that URLs have *path*s and not *pathname*s.